worker/task.go

```diff
@@ -55,6 +55,7 @@ func NewTaskRun(
 		Expires:  claim.taskClaim.Task.Expires,
 	}
 	ctxt, ctxtctl, err := runtime.NewTaskContext(tp, info)
+	ctxt.LogEntry = log
```

Comment:
Oh, maybe I'm wrong here... It probably already has that property...

Maybe a property: `kind:task-log`